portafly/src/i18n/locales/en/login.yml

```diff
@@ -0,0 +1,14 @@
+---
+page_title: Login | Portafly
+page_title_description: Log in to your account
+body_title: Login
+body_title_description: Login page
```

Comment:
```suggestion
body_title_description: Body title of the Login page
```

Reply:
I think it pretty much says it all in the name of the key. no?